### ISSUE TITLE
I don't see the state of the instances on each node

### ISSUE THREAD

**Issue by tthiebaut** [freddenis/oracle-scripts#10]
Hi Fred,

on exa X7 (19.1.1.2.2.0.0) with the grid in 18.5, I don't see the state of the instances on each node:

```
                Cluster s2x7 is a X7-2 Quarter Rack HC 10TB

      Listener   |      Port     |         db01        |         db02        |     Type     |
  -------------------------------------------------------------------------------------------
   LISTENER      | TCP:1521      |          -          |          -          |   Listener   |
   LISTENER_SCAN1| TCP:1521      |          -          |          -          |     SCAN     |
   LISTENER_SCAN2| TCP:1521      |          -          |          -          |     SCAN     |
   LISTENER_SCAN3| TCP:1521      |          -          |          -          |     SCAN     |
  -------------------------------------------------------------------------------------------

         DB      |    Version    |         db01        |         db02        |    DB Type   |
  -------------------------------------------------------------------------------------------
   colfle1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   colfle2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   colfpe2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   datawe1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (P)   |
   db1           | 12.1.0.2  (2) |          -          |          -          |    RAC (P)   |
   db2           | 11.2.0.4  (1) |          -          |          -          |    RAC (P)   |
   dwhre1x7_s2   | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   dwjdee1x7_s2  | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   fime2x7_s2    | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   hr9ce1x7_s2   | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   hr9cprx7_s2   | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   hr9e1x7_s2    | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   hr9e2x7_s2    | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   hr9refx7_s2   | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   jde92e1x7_s2  | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   jde92e2x7_s2  | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   jde92etx7_s2  | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   jde92m2x7_s2  | 12.1.0.2  (2) |          -          |          -          |    RAC (P)   |
   jdee1x7_s2    | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   jdee2x7_s2    | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   liaise1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   liaise2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   liaism2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   mutise1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   ofme11x7_s2   | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   ofme15x7_s2   | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   ofme1x7_s2    | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   ofme21x7_s2   | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   ofme2x7_s2    | 12.1.0.2  (2) |          -          |          -          |    RAC (S)   |
   refupe2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   refute1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   refute2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   wlrefe1x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
   wlrefe2x7_s2  | 11.2.0.4  (1) |          -          |          -          |    RAC (S)   |
  -------------------------------------------------------------------------------------------

  ORACLE_HOME references listed in the Version column ("''" means "same as above")

         1 : /u01/app/oracle/product/11.2.0.4/dbhome_1  oracle oinstall
         2 : /u01/app/oracle/product/12.1.0.2/dbhome_1    ''      ''
```

**Comment by freddenis** [comment]
Hi,

Could you please do:
./rac-status.sh -o file_for_fred.txt

and send me the "file_for_fred.txt" file by email ? fred.denis3@gmail.com

I could find and fix the issue then.

Thanks,

Fred

**Comment by freddenis** [comment]
Fixed in version 20190524

**Comment by freddenis** [comment]
> Fixed in version 20190524

Closing the issue.
